crates/sui-framework/packages/deepbook/sources/clob_v2.move

```diff
@@ -295,9 +295,31 @@ module deepbook::clob_v2 {
         ctx: &mut TxContext,
     ) {
         assert!(coin::value(&creation_fee) == FEE_AMOUNT_FOR_CREATE_POOL, EInvalidFee);
-        create_pool_<BaseAsset, QuoteAsset>(
+        create_customized_pool<BaseAsset, QuoteAsset>(
+            tick_size,
+            lot_size,
             REFERENCE_TAKER_FEE_RATE,
             REFERENCE_MAKER_REBATE_RATE,
+            creation_fee,
+            ctx,
+        );
+    }
+
+    // Function for creating pool with customized taker fee rate and maker rebate rate.
+    // The taker_fee_rate should be greater than or equal to the maker_rebate_rate, and both should have a scaling of 10^9.
```

Comment:
should we assert this? Also ok if you don't since people just won't use the pool

Reply:
We have asserted this inside the internal function "create_pool_".